lib/membrane_ffmpeg_swscale/scaler/scaler.ex

```diff
@@ -33,33 +33,33 @@ defmodule Membrane.FFmpeg.SWScale.Scaler do
               ],
               use_shm?: [
                 type: :boolean,
```

Comment:
The field type should be replaced with `spec`
```suggestion
                spec: boolean(),
```

Same goes for other options (GH does not allow me to comment on them)